chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+
+***Impact of confidence level:*** the greater the confidence level, the wider the confidence intervals.
+
+  * *Intuition*: a greater confidence level means the confidence interval needs to contain the true population parameter more often, and thus needs to be wider to ensure this.
+  
+--
+
+***Impact of sample size:*** the greater the confidence level, the wider the confidence intervals.
+
+  * *Inutuition*: a larger sample size leads to less sampling variation and therefore a narrower boostrap distribution, which in turn leads to thiner confidence intervals.
+  
+---
+
+# From Confidence Intervals to Hypothesis Testing
+
+* *Confidence intervals* can be thought of as an extension of *point estimation*.
+
+--
+
+* What if we want to ***compare*** a sample statistic for two groups?
+  
+  * *Example*: differences in average wages between men and women. Are the observed differences ***significant***?
+  
+--
+
+* These comparisons are the realm of ***hypothesis testing***.
+
+--
+
+* Just like confidence intervals, hypothesis tests are used to make claims about a population based on information from a sample.
+
+* However, we’ll see that the framework for making such inferences is slightly different.
+
+---
+
+layout: false
+class: title-slide-section-red, middle
+
+# Hypothesis Testing
+
+---
+layout: true
+
+<div class="my-footer"><img src="../img/logo/ScPo-shield.png" style="height: 60px;"/></div> 
+
+---
+
+# Is There Gender Discrimination In Promotions?
+
+* We will use data from an [article](https://pdfs.semanticscholar.org/39f6/d40e907ff08af4ddd3280c2ceee55ee1ddb6.pdf) published in the *Journal of Applied Psychology* in 1974 which investigated whether female employees at banks were discriminated against.
+
+* 48 (male) supervisors were given *identical* candidate CVs, differing only with respect to the first name, which was male or female.
+
+  * Each CV was "*in the form of a memorandum requesting a decision on the promotion of an employee to the position of branch manager.*"
+
+--
+
+* ***Hypothesis*** we want to test: *Is there gender discrimination?*
+
+--
+
+.pull-left[
+* The data from the experiment are provided in the `promotions` dataset from the `moderndive` package.
+]
+
+--
+
+.pull-right[
+```{r}
+library(moderndive)
+head(promotions)
+```
+]
+
+---
+
+# Evidence of Discrimination?
+
+.pull-left[
+How many men and women were offered a promotion (and not)?
+
+```{r}
+promotions %>% 
+  group_by(gender, decision) %>% 
+  tally() %>%
+  mutate(percentage = 100 * n / sum(n))
+```
+
+There is a ***29.2 percentage points difference*** in promotions between men and women!
+]
+
+--
+
+.pull-left[
+```{r,echo= FALSE, fig.height=6}
+ggplot(promotions, aes(x = gender, fill = decision)) +
+  geom_bar(width = 0.75) +
+  labs(x = "Gender of name on resume",
+       y = "Frequency") +
+  labs(title = "Promotion decision") +
+  theme_bw(base_size = 20) +
+  theme(legend.position = "top")
+```
+]
```

Comment:
that bracket needs to come after the last paragraph. you can't have the `--` in the pull-right.

Reply:
should be a `pull-right` my bad. interestingly it shows in the right sidebar.